src/jvmMain/kotlin/com/sdercolin/vlabeler/video/VideoMain.kt

```diff
@@ -0,0 +1,39 @@
+package com.sdercolin.vlabeler.video
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import com.sdercolin.vlabeler.audio.PlayerState
+import com.sdercolin.vlabeler.env.Log
+import com.sdercolin.vlabeler.util.toMillisecond
+import com.sdercolin.vlabeler.util.or
+
+@Composable
+fun VideoMain(
+    videoState: VideoState,
+    playerState: PlayerState
+) {
+    require(videoState.videoPath != "") { "video path not located" }
```

Comment:
Just get rid of that. Because `fun Video()` is called only after `find()` (find `videoPath`), the responsibility to check `videoPath` is in `find()` now.